README.md

```diff
@@ -14,6 +14,7 @@ Contents
 --------
 
 - [Usage](#usage)
+  - [Observing WiFi Connection](#observing-wifi-access-points)
```

Comment:
It should be [Observing WiFi Connection](#observing-wifi-connection) to make link correct